com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.RenderGraph.cs

```diff
@@ -317,9 +317,7 @@ TextureHandle DynamicExposurePass(RenderGraph renderGraph, HDCamera hdCamera, Te
                     {
                         passData.source = builder.ReadTexture(source);
                         passData.parameters = PrepareApplyExposureParameters(hdCamera);
-                        RTHandle prevExp;
-                        GrabExposureHistoryTextures(hdCamera, out prevExp, out _);
-                        passData.prevExposure = builder.ReadTexture(renderGraph.ImportTexture(prevExp));
+                        passData.prevExposure = renderGraph.ImportTexture(GetPreviousExposureTexture(hdCamera));
```

Comment:
@FrancescoC-unity An alternative fix would be to have
`passData.prevExposure =nextExposureHandle;`
This was my first fix, but in the end I have used the GetPreviousExposureTexture call because it was also used in the non-rendergraph version.

Reply:
GetPreviousExposureTexture is the correct thing to call as it setups the history system correctly.

Reply:
However isn't that grabbed already by GrabExposureRequiredTextures ? If I read the code right, no need to do the get again and perform the import again, you should have the handle in prevExposureHandle

Reply:
If I use the prevExposureHandle, then the ApplyExposure pass will run with the default EV100 exposure texture and this creates a very bright flashing in the first frame (which we were not getting in the non-rendergraph path). This is also the case if we don't change anything.

My understanding (and what we were doing in the non-render graph case) is that we have to use the exposure texture that we have just computed with the code in the previous lines (the dynamic-exposure pass). I get this behavior with the GetPreviousExposureTexture or if I use the nextExposureHandle directly (this is where the dynamic-exposure pass stores the result), but not with the prevExposureHandle.

Reply:
Gotcha, my memory is fuzzy here, but if it was mimicking what was in the non-rendergraph path, all good 👍